intro-to-linux/PITCHME.md

```diff
@@ -4,19 +4,48 @@
 
 --- 
 ## What is Linux ?
-Linux is an open source operating system (OS).
-Every Linux-based OS involves the Linux kernel
+Linux is an open source operating system (OS). 
+
+A good start to the class would be asking the students if they know what Linux is, although they may be quiet at first, we found that the more engaging a class is, the more keen they are to learn too.
+
+* What are other famous open source operating systems?
+* What is an operating system?
+* Do they use an OS?
+
+
+Every Linux-based OS involves the Linux kernel.
+
+THat's the main component of the Linux operating system, communicates between a computer's hardware and its processes.
+
+A nice analogy to explain the role of the kernel:
+
+
+```
+The kernel is a busy personal assistant for a powerful executive (the hardware).
+
+It’s the assistant’s job to relay messages and requests (processes) from employees and the public (users) to the executive, to remember what is stored where (memory), and to determine who has access to the executive at any given time and for how long.
+```
 
 
-+++
 ## Linux and Open source
-Anyone can run, study and modify and redistribute the linux source code
+Anyone can run, study and modify and redistribute the Linux source code.
+
 
-+++
 ## Linux Distributions
+
+Linux distribution = complete Linux system package
+
+Different distros can be customized for specific user gorups, for example software enginners or audio/video editors
+
```

Comment:
Maybe something along these lines: 

The reason why different distros exist is because Linux is an open and free system. This means that people who have different needs and different philosophies can use and develop Linux - each according to their own needs.

For example, some people like to have a large corporation support them, and are prepared to pay for that support. Other people don't want to pay, but want a system that is stable and doesn't change unless they choose to update it. There are also people who are excited by what the technology can do, and want to try out new things - and don't mind if things break occasionally as long as they have the latest and greatest.

None of those viewpoints is right or wrong - they're just different ways to look at the world, and Linux (and Free and Open Source in general) can accommodate all of those different perspectives.

The right to use, change, modify and share the code is a fundamental freedom. The only thing that is asked of you in return is that you respect the freedom of others to do as they want to with the code, and any changes that you make to it.